.github/workflows/e2e-android.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           export PATH=$ANDROID_SDK_ROOT/platform-tools:$PATH
           export PATH=$ANDROID_SDK_ROOT/emulator:$PATH
           cd packages/mobile
-          yarn run test:e2e:android -w 3
+          yarn run test:e2e:android -w 1
```

Comment:
is this back to 1 on purpose? 🥲

Reply:
Was attempting to see if that was the issue, but I've bumped it back up to 3.